composer.json

```diff
@@ -14,5 +15,10 @@
         "psr-4": {
             "Neos\\RedirectHandler\\": "Classes"
         }
+    },
+    "extra": {
+        "branch-alias": {
+            "dev-master": "1.0.x-dev"
+        }
```

Comment:
`>=3.2` would allow it to be installed with master as well allowing just one branch until actually necessary code wise.. maybe we should do the same for SEO and Analytics

that just means we need to always have a compatible version with the latest master